core/string/ustring.h

```diff
@@ -285,8 +286,11 @@ class String {
 	int find(const char *p_str, int p_from = 0) const; ///< return <0 if failed
 	int find_char(const char32_t &p_char, int p_from = 0) const; ///< return <0 if failed
 	int findn(const String &p_str, int p_from = 0) const; ///< return <0 if failed, case insensitive
+	int findn(const char *p_str, int p_from = -1) const; ///< return <0 if failed
```

Comment:
```suggestion
	int findn(const char *p_str, int p_from = 0) const; ///< return <0 if failed
```
Incorrectly copied argument, won't work with default argument

Reply:
Thank you!